code/gui/mainwindow.ui

```diff
@@ -34,7 +34,14 @@
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
-        <widget class="QLineEdit" name="move_command_edit"/>
+        <widget class="QLineEdit" name="move_command_edit">

```

Comment:
Ignoring UI stuff for now, assuming this works and was auto-populated by Qt or equivalent. Do we have images showing what arrow is pressed right now? If not, are we planning on adding them in?